src/App.js

```diff
@@ -1,11 +1,18 @@
-import './App.css';
+import styles from "App.module.css"
```

Comment:
```suggestion
import "app.css"
```
Let's also rename `App.module.css` to `app.css`